app/main.py

```diff
@@ -1,2 +1,18 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    return phrase.lower().count(letter.lower())
+     """
+     Implement count_occurrences function:
+
+    It takes a phrase and a letter and calculates the number of times
+    the letter appears in the phrase. The function is case insensitive.
+    count_occurrences("letter", "t") == 2
+    count_occurrences("abc", "a") == 1
+    count_occurrences("abc", "d") == 0
+    count_occurrences("ABC", "a") == 1
+    :param phrase: phrase to count in it
+    :param letter: letter to find occurrences of it
+    :return: count occurrences of letter in phrase
+    """
+     
+   return phrase.lower().count(letter.lower())
+  
```

Comment:
My bad, you had everything fine with the last empty line. Now you have two of them, so please remove one, sorry)